### DISCUSSION TITLE
custom-timestamps-fields-in-elixir

### DISCUSSION THREAD

**Discussion by giscus** [bartoszgorka/bartoszgorka.github.io#118] [Announcements]
# custom-timestamps-fields-in-elixir

Ecto Schema is very easy to adapt to our needs. Besides changing the type of the primary key, you can also change the type and names of the fields created by the timestamps() macro.

https://bartoszgorka.com/custom-timestamps-fields-in-elixir

**Comment by ksaitor** [comment]
omg! thanks for sharing this! saved my life

**Reply by bartoszgorka** [reply]
Amazing, thank you for sharing it!
